From 60bdfe1e8fa5761dc27c6d4a091e14555259d0c9 Mon Sep 17 00:00:00 2001
From: Yasufumi Ogawa <ogawa.yasufumi@lab.ntt.co.jp>
Date: Thu, 16 May 2019 04:14:16 +0900
Subject: [PATCH 4/6] shared/sec: rename func for parsing lcore ID

This update is to rename function for parsing lcore ID
`decode_component_core_value()` in which the lcore vaule is not
understandable to `parse_comp_lcore_id()`.

Signed-off-by: Yasufumi Ogawa <ogawa.yasufumi@lab.ntt.co.jp>
---
 src/shared/secondary/spp_worker_th/cmd_parser.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/shared/secondary/spp_worker_th/cmd_parser.c b/src/shared/secondary/spp_worker_th/cmd_parser.c
index cf6c4a1..b03f9b8 100644
--- a/src/shared/secondary/spp_worker_th/cmd_parser.c
+++ b/src/shared/secondary/spp_worker_th/cmd_parser.c
@@ -350,14 +350,14 @@ parse_comp_name(void *output, const char *arg_val,
 	return SPP_RET_OK;
 }
 
-/* decoding procedure of core id for component command */
+/* Parse given lcore ID of `arg_val` of component. */
 static int
-decode_component_core_value(void *output, const char *arg_val,
+parse_comp_lcore_id(void *output, const char *arg_val,
 				int allow_override __attribute__ ((unused)))
 {
 	struct sppwk_cmd_comp *component = output;
 
-	/* "stop" has no core ID parameter. */
+	/* Parsing lcore is required only for action `start`. */
 	if (component->wk_action != SPPWK_ACT_START)
 		return SPP_RET_OK;
 
@@ -808,7 +808,7 @@ cmd_ops_list[][SPPWK_MAX_PARAMS] = {
 		{
 			.name = "core",
 			.offset = offsetof(struct spp_command, spec.comp),
-			.func = decode_component_core_value
+			.func = parse_comp_lcore_id
 		},
 		{
 			.name = "component type",
-- 
2.17.1

